components/tooltip/tooltip.js

```diff
@@ -310,12 +310,13 @@ class Tooltip extends RtlMixin(LitElement) {
 				border-radius: ${contentBorderRadius}px;
 				box-sizing: border-box;
 				max-width: 17.5rem;
-				min-height: 2.1rem;
+				min-height: 1.95rem;
```

Comment:
Now that the line height is `2px` smaller and the padding `1px` reduced, we needed to reduce the `min-height` by `3px` to compensate.